modules/creating-custom-ssl-certs-config-bundle.adoc

```diff
@@ -0,0 +1,238 @@
+:_content-type: PROCEDURE
+[id="creating-custom-ssl-certs-config-bundle"]
+= Creating a custom SSL/TLS configBundleSecret resource
+
+After creating your custom SSL/TLS certificates, you can create a custom `configBundleSecret` resource for {productname-ocp}, which allows you to upload `ssl.cert` and `ssl.key` files.
+
+.Prerequisites
+
+* You have base64 decoded the original config bundle into a `config.yaml` file.
+* You have generated custom SSL certificates and keys.
+
+.Procedure
+
+. Create a new YAML file, for example, `custom-ssl-config-bundle-secret.yaml`:
++
+[source,terminal]
+----
+$ touch custom-ssl-config-bundle-secret.yaml
+----
+
+. Create the `custom-ssl-config-bundle-secret` resource.
+
+.. Create the resource by entering the following command:
++
+[source,terminal]
+----
+$ oc -n <namespace> create secret generic custom-ssl-config-bundle-secret \
+  --from-file=config.yaml=<config.yaml> \ <1>
+  --from-file=ssl.cert=<ssl.cert> \ <2>
+  --from-file=extra_ca_cert=<ca.cert> \ <3>
+  --from-file=ssl.key=<ssl.key> \ <4>
+  --dry-run=client -o yaml > custom-ssl-config-bundle-secret.yaml
+----
+<1> Where `<config.yaml>` is your `base64 decoded` `config.yaml` file.
+<2> Where `<ssl.cert>` is your `ssl.cert` file.
+<3> Optional. The `--from-file=extra_ca_cert=<ca.cert>` field allows {productname} to recognize custom Certificate Authority (CA) files. If you are using LDAP, OIDC, or another service that uses custom CAs, you must add them via the `extra_ca_cert` path.
+<4> Where `<ssl.key>` is your `ssl.key` file.
+
+. Optional. You can check the content of the `custom-ssl-config-bundle-secret.yaml` file by entering the following command:
++
+[source,terminal]
+----
+$ cat custom-ssl-config-bundle-secret.yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+apiVersion: v1
+data:
+  config.yaml: QUxMT1dfUFVMTFNfV0lUSE9VVF9TVFJJQ1RfTE9HR0lORzogZmFsc2UKQVVUSEVOVElDQVRJT05fVFlQRTogRGF0YWJhc2UKREVGQVVMVF9UQUdfRVhQSVJBVElPTjogMncKRElTVFJJQlVURURfU1R...
+  ssl.cert: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUVYakNDQTBhZ0F3SUJBZ0lVTUFBRk1YVWlWVHNoMGxNTWI3U1l0eFV5eTJjd0RRWUpLb1pJaHZjTkFRRUwKQlFBd2dZZ3hDekFKQmdOVkJBWVR...
+  extra_ca_cert:LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUVYakNDQTBhZ0F3SUJBZ0lVTUFBRk1YVWlWVHNoMGxNTWI3U1l0eFV5eTJjd0RRWUpLb1pJaHZjTkFRRUwKQlFBd2dZZ3hDe...
```

Comment:
Same as above. The key name must be in the form `extra_ca_cert_{NAME-OF-CERT}.crt`.